guides/createMakerAsk.md

```diff
@@ -1,57 +1,71 @@
-:warning: These code snippets are just examples and the data should never be used as is :warning:
+---
+title: "Manually creating maker ask"
+group: Guides
+---
+
+> [!WARNING]
+>
+> These code snippets are just examples and the data should never be used as is
 
 # How to create a maker ask order
 
-The code snippet below is an example of how to create a maker ask using the `@looksrare/sdk-v2` library.
+The code snippet below is an example of how to create a maker ask using the `@hypercerts-org/marketplace-sdk` library.
 
 The main steps are:
 
-1. Initialize a LooksRare class instance by providing the chain id, [RPC provider](https://docs.ethers.io/v5/api/providers/) and a [signer](https://docs.ethers.io/v5/api/signer/).
+1. Initialize a HypercertExchangeClient class instance by providing the chain id, [RPC provider](https://docs.ethers.io/v6/api/providers/) and a [signer](https://docs.ethers.org/v6/api/providers/#Signer).
 2. Use the `createMakerAsk` method to create a maker ask with the parameters of your order.
 3. Check and grant necessary approvals for transferring assets.
 4. Sign the maker ask order with `signMakerOrder` method.
 
-> The `orderNonce` has to be retrieved via our Public API, see [get order nonce](https://looksrare.dev/v2/reference/getordernonce).
+> The `orderNonce` has to be retrieved via our Public API, see [get order nonce](https://api.hypercerts.org/spec/#/Marketplace/UpdateOrderNonce).
 
 Here is an example:
 
 ```ts
 import { parseEther } from "ethers";
-import { LooksRare, ChainId, CollectionType, StrategyType } from "@looksrare/sdk-v2";
+import { HypercertExchangeClient, ChainId, CollectionType, StrategyType } from "@hypercerts-org/marketplace-sdk";
+
+const hypercertExchangeClient = new HypercertExchangeClient(ChainId.OPTIMISM, provider, signer);
+const orderNonce = await hypercertExchangeClient.api.fetchOrderNonce({
+  address, // Your address
+  chainId: ChainId.OPTIMISM, // Chain ID
+});
 
-const lr = new LooksRare(ChainId.MAINNET, provider, signer);
+// The ID of the fraction to be sold
+const fractionId = 13601086205829910384631083059047775411896320n;
 
-const { maker, isCollectionApproved, isTransferManagerApproved } = await lr.createMakerAsk({
-  collection: "0x0000000000000000000000000000000000000000", // Collection address
-  collectionType: CollectionType.ERC721,
+const { maker, isCollectionApproved, isTransferManagerApproved } = await hypercertExchangeClient.createMakerAsk({
+  collection: "0x123", // Collection address
```

Comment:
Same pet peeve as in other docs, actually use addresses because everytime I read this I think the example is incorrect